### ISSUE TITLE
Stuck Ship

### ISSUE THREAD

**Issue by ghan210** [MaFi-Games/Captain-of-Industry-issues#1372]
### Issue description

I wanted to expand my cargo depot so I built a new ship yard in order to 'move' it. I selected make primary on the new ship yard however my ship was out on a mission and is now stuck at an island and won't return and I have 2x shipyards with the new ship yard indicating it is currently being reassigned. The ship itself is displayed as at an island but when I select deliver cargo is says already present or on its way. I seem stuck in a loop and am not sure what to do.

### Reproduction steps

[shipstucksave_24-03-17_18-39-36.zip](https://github.com/MaFi-Games/Captain-of-Industry-issues/files/14630797/shipstucksave_24-03-17_18-39-36.zip)

### Game version and build number

v0.5.5d b260

### Operating system

Windows

**Comment by KptKosmit91** [comment]
I just checked, and I don't see anything wrong. You first need to send the ship back to the Home Island to be able to send it to the settlement with the cargo, after the cargo has been loaded

**Comment by poltime** [comment]
LOL. Open the map, click on the home island, press the "GO HOME" button.

**Comment by ghan210** [comment]
Yes. I get that I needed to select 'go home' and yes I had that button available.
But for reasons I couldn't understand, despite clicking it many times my ship just remained at the distant island 'already present or on its way'.
In the end I deleted the new ship yard and upon the completion of that action my ship was then able to return 'home' after selecting the option once more.
As an experiment I closed the game down, restarted & reloaded the save file (with two ship yards) and yes - the go home button worked straight away. So I don't know what happened.
